src/AutoRest.CSharp/Common/Generation/Writers/RequestWriterHelpers.cs

```diff
@@ -28,13 +29,13 @@ namespace AutoRest.CSharp.Generation.Writers
 {
     internal static class RequestWriterHelpers
     {
-        public static void WriteRequestCreation(CodeWriter writer, RestClientMethod clientMethod, string methodAccessibility, ClientFields? fields, string? responseClassifierType, bool writeSDKUserAgent, IReadOnlyList<Parameter>? clientParameters = null)
+        public static void WriteRequestCreation(CodeWriter writer, RestClientMethod clientMethod, ClientFields? fields, string? responseClassifierType, bool writeSDKUserAgent, IReadOnlyList<Parameter>? clientParameters = null)
```

Comment:
parameter `methodAccessibility` is removed here because in all the places we are invoking this method, we always pass in the value "internal".
Therefore for simplicity, I removed it and hard-coded it in the method.

We should schedule a refactor to this method to use the expressions to build the method instead of using strings, and when we do that, we could add the parameter back via `MethodSignatureModifiers`

Reply:
Can you link a tracking issue for this?

Reply:
I believe it is tracking here by Alexander: https://github.com/Azure/autorest.csharp/issues/3768